features/javascript/request_api.feature

```diff
@@ -35,7 +35,7 @@ Feature: Request validation
   Scenario: validate
     When you call:
     """
-    gavel.validate(request, expected);
+    gavel.validate(expected, real);
```

Comment:
Most similiar assertion (test) style functions both in NodeJS (https://nodejs.org/api/all.html#assert_assert_equal_actual_expected_message), chai (https://www.chaijs.com/api/bdd/#method_fail), etc use the terminology "expected" and "actual".

Would it make sense to rename `real` to `actual` for consistency?

Reply:
Absolutely! `real` sounded off to me, but spent weeks on Gavel I've got used to it.
I feel `expected` and `actual` are more descriptive.

Reply:
I believe the change of the naming would need to go through the whole stack (gavel, dredd, apiary). Something tells me that we're inconsistent and we use the word "actual" somewhere instead of "real".

I have no strong preference. We inherited the naming from our predecessors (dredd and gavel creators) and I'm fine going with anything that seems more "standard" and understandable by the most users.

Reply:
We decided this will probably come as a separate change. We should file an issue for that.

Reply:
Tracked here https://github.com/apiaryio/gavel-spec/issues/45.